### ISSUE TITLE
Nx HMR not working with styled

### ISSUE THREAD

**Issue by n8sabes** [qwikifiers/qwik-nx#60]
**Describe the bug**
Hot module reload not working with styled updates making it very difficult to develop in Nx. Requires full restart for **EVERY** edit.

**To Reproduce**

1. See the video: [HERE](https://www.loom.com/share/99dd5c2c6aa7441f99d16b7779063d7a)
Sorry there is no audio, but the first example is a clean vanilla non-Nx qwik build. The second is exactly the same, a clean vanilla Nx qwik build. 
2. Clone the repo [HERE](https://github.com/n8sabes/nx-styled-ve)
3. Nx serve client
4. Change styles in `FancyHeader/styles.css.ts`

**Expected behavior**
Recompile appears to happen, or partially happen, but no updates occur in the browser.

**Comment by shairez** [comment]
that's weird indeed

my guess is that it's a bug in the nx/vite plugin

plus it only happens for the `styled` way right?  with normal css it still works right?

**Comment by n8sabes** [comment]
@shairez, You are correct. It only happens in Vanilla Extract and Styled Vanilla Extract.

Please see video: [HERE](https://www.loom.com/share/9a6c4d5f2bde47939ff75ecc77094c66)
